pkgs/by-name/ps/ps3-disc-dumper/update.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=./. -i bash -p curl jq common-updater-scripts
+set -eo pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+new_version="$(curl -s "https://api.github.com/repos/13xforever/ps3-disc-dumper/releases?per_page=1" | jq -r '.[0].tag_name | ltrimstr("v")')"
+old_version="$(sed -nE 's/\s*version = "(.*)".*/\1/p' ./package.nix)"
+if [[ "$new_version" == "$old_version" ]]; then
+    echo "Up to date"
+    exit 0
+fi
+
+cd ../../../..
```

Comment:
nit: I'd recommend using `pushd` and `popd` instead of depending on the location of the file in nixpkgs.